### PR TITLE
Meeting layout and check-in updates

### PR DESCRIPTION
**Pull request by ackernaut**
This mainly addresses #627 (Updates vertical real estate) and #717 (improving Check-In round):

- Merges avatar and name into meeting phase prompts (combining two groups of content into one).
- Works at simplifying check-in round (still designing).
- Bye, bye Progress Bar as we are showing navigation state with the Team Avatars.
- Utilizes LTR layout on Updates phase for taller project columns.

Plus, sneaking in some other things:

- Going to work in a little depth with some shadows.
- Going to add badges to project columns (and improve the styles: use less borders and dashed vertical borders)

### CODE REVIEW

**Comment by ackernaut** [comment]
This is a currently a WIP. As of now, it would be helpful to get some feedback on the current state of UX (then we can dig into code as we settle on the UX). @jordanh @mattkrick want first stabs at seeing what the meeting is like?

**Comment by ackernaut** [comment]
Current WIP screenshots (1280x720):

screenshot 2017-04-04 11 56 46
screenshot 2017-04-04 11 56 58
screenshot 2017-04-04 11 57 07
screenshot 2017-04-04 11 57 16

**Comment by ackernaut** [comment]
Latest screens (1280x720):

screenshot 2017-04-05 13 46 22
screenshot 2017-04-05 13 46 35
screenshot 2017-04-05 13 46 41
screenshot 2017-04-05 13 46 52
screenshot 2017-04-05 13 47 02
screenshot 2017-04-05 13 47 06

**Comment by ackernaut** [comment]
Ok, folks, this is ready for review:

1. If the Facilitator simply presses `enter` or `right` through the Check-In round, users do not get a `check` or `minus` on their avatar badge and they show up as absent in the Summary view. I was wondering if we could check if they are checked in or not, and if not they get marked as present if the facilitator presses enter/right without clicking Present/Not Present. (I’m thinking we can do this in a separate, smaller PR) but I can be persuaded.
2. There is an error in console about the styles object being required and undefined in MeetingUpdates, but I removed `required` and it still shows the error. Not sure what to do there.
3. Running `npm run lint` should be clean.

Thanks @tayamueller for jamming on Check-In UX with me.

**Comment by jordanh** [comment]
Awesome work. Looking now!

Can you explain #1 a different way? I'm not sure I am following you...

**Comment by ackernaut** [comment]
@jordanh here is another way to explain it: #884

File: src/universal/components/Badge/Badge.js [hunk 1]
```diff
@@ -31,11 +39,12 @@ Badge.defaultProps = {
   value: 0
 };
 
+const boxShadow = '1px 1px 2px rgba(0, 0, 0, .5)';
```
Comment (mattkrick) [inline src/universal/components/Badge/Badge.js]: should boxShadow live in `ui`? I feel like we'll wanna re-use the same 5 or so box shadows.

Comment (ackernaut) [inline src/universal/components/Badge/Badge.js]: Yeah, I agree and will move them into `ui.js` as the patterns normalize. Some are already there.

**Pull request merged**